client/packages.go

```diff
@@ -72,3 +72,15 @@ func (client *Client) Packages() (map[string]Package, error) {
 
 	return packages, nil
 }
+
+// Package returns details of a single Package
+func (client *Client) Package(pkgID string) (Package, error) {
```

Comment:
Might be easier for us to cover that online, but as a conversation-starter, when we have a Person at hand, we generally want that value to be held as a reference. This is the one person we're talking about, and we don't want that object to be copied on every operation. On the other side, when we handle dates and times, for example, it makes most sense to use value semantics, because a given time has no sense of identity. It's just a value we pass around.

Happy to extend the topic online.